### PR TITLE
Updated 1 header in slates readme, and made the links requested in th…

### PR DESCRIPTION
**Pull request by jarumihooi**
…e template md into separate lines for ease of template use later.

To conform to template

### CODE REVIEW

File: template-for-readme.md [hunk 1]
```diff
@@ -20,13 +20,13 @@
...
-(link for user manual, if applicable)
-(Any intro needed for tool)
-
+(link for download, if applicable)  
+(link for source code, if applicable)  
+(link for installation instructions, if applicable)  
+(link for documentation, if applicable)  
+(link for user manual, if applicable)  
+(Any intro needed for tool)  
+
```
Comment (keighrim) [inline template-for-readme.md:29]: I believe these are probably nothing, but just wanted to make sure that these (non-)changes are not related to any changes you might have made to address line break problem. Or, are they?

Comment (jarumihooi) [reply template-for-readme.md:29]: I simply added two spaces to make these all appear on different lines so that in the future it is easier to grab and replace with the real link text. 

(Has nothing to do with file-formatting, LF etc)

**Review by keighrim — APPROVED** [review]
